x-pack/plugins/osquery/server/routes/saved_query/create_saved_query_route.ts

```diff
@@ -30,6 +30,13 @@ export const createSavedQueryRoute = (router: IRouter) => {
 
       const { id, description, platform, query, version, interval } = request.body;
 
+      try {
+        await savedObjectsClient.get(savedQuerySavedObjectType, id);
```

Comment:
this works, but feels gross. is there a cleaner way to do this? i couldn't find a method to easily check for existence of an id.